Client/WebFilters/SafeBrowsing/SafeBrowsing.swift

```diff
@@ -47,20 +47,14 @@ class SafeBrowsing {
     }
     
     private func fillDomainList() {
-        DispatchQueue.global(qos: .background).async {
-            var newList = Set<String>()
-            
-            self.listNames.forEach {
-                if let list = self.openList(withName: $0.name) {
-                    newList.formUnion(self.parse(list, regex: $0.regex))
-                }
-            }
-            
-            DispatchQueue.main.async {
-                log.info("Safe browsing list was filled with \(newList.count) records")
-                self.domainList = newList
+        var newList = Set<String>()
+
+        self.listNames.forEach {
```

Comment:
I know you didnt make a change in this loop but since we are altering the code here, can we use map instead of foreach and assigning the result.